src/config/texts.js

```diff
@@ -1548,7 +1550,7 @@ export const texts = {
     reminder: 'Erinnerungen',
     reminderTime: 'Zu welcher Uhrzeit möchten Sie benachrichtigt werden?',
     sameDay: 'selber Tag',
-    save: 'Speichern',
+    save: 'Änderungen Speichern',
```

Comment:
speichern

Reply:
Ich habe es direkt von Figma kopiert. 🤷‍♂️

<img width="437" alt="image" src="https://github.com/user-attachments/assets/c976a7ff-b7a3-4bba-b6d1-060eef1bcacf" />

Reply:
ok then please comment in figma as well to fix the typo there